Marlin/configuration_store.cpp

```diff
@@ -187,20 +187,21 @@ void Config_StoreSettings()  {
   EEPROM_WRITE_VAR(i, planner.max_e_jerk);
   EEPROM_WRITE_VAR(i, home_offset);
 
+  uint8_t dummy_uint8 = 0;
   uint8_t mesh_num_x = 3;
   uint8_t mesh_num_y = 3;
   #if ENABLED(MESH_BED_LEVELING)
     // Compile time test that sizeof(mbl.z_values) is as expected
     typedef char c_assert[(sizeof(mbl.z_values) == (MESH_NUM_X_POINTS) * (MESH_NUM_Y_POINTS) * sizeof(dummy)) ? 1 : -1];
     mesh_num_x = MESH_NUM_X_POINTS;
     mesh_num_y = MESH_NUM_Y_POINTS;
-    EEPROM_WRITE_VAR(i, mbl.active);
+    dummy_uint8 = mbl.status & 0x01; // Do not save 'is active'
```

Comment:
Instead of using bit numbers in raw form, it is a best practice to give names to bits and other special values. It makes the code more self-documenting and reduces the need to know the meaning of mystery bit numbers or values. So, you should give the bits names, for example:
```cpp
enum { MBL_ACTIVE = 0, MBL_HAS_MESH };
```
…and then use them like so…
```cpp
dummy_uint8 = mbl.status & _BV(MBL_ACTIVE);
```